libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -2036,9 +2023,8 @@ def write_PWM_config(self, f, ordered_timers):
         pwm_out = []
         # start with the ordered list from the dma resolver
         pwm_timers = ordered_timers
-        has_bidir = False
```

Comment:
```has_bidir = False``` is removed, is that confirmed to not change behavior?

Reply:
@magicrub - I think its unused - part of the flake8 cleanup which I think should have been in a separate PR

Reply:
It's in a separate commit, "minor bug fixes".

No bug fixed materially affects anything except flake8 cleanliness, this rolling these into the one PR.